java-checks/src/test/java/org/sonar/java/checks/naming/BadTestClassNameCheckTest.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * SonarQube Java
+ * Copyright (C) 2012-2016 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.java.checks.naming;
+
+import org.junit.Test;
+import org.sonar.java.checks.verifier.JavaCheckVerifier;
+
+public class BadTestClassNameCheckTest {
+  @Test
+  public void test() {
+    BadTestClassNameCheck check = new BadTestClassNameCheck();
+    JavaCheckVerifier.verify("src/test/files/checks/naming/BadTestClassNameCheck.java", check);
+    // play a second time with same check to reuse pattern
+    JavaCheckVerifier.verify("src/test/files/checks/naming/BadTestClassNameCheck.java", check);
```

Comment:
Only to cover the lazy initialization of the pattern ([BadTestClassNameCheck.java, L58](https://github.com/SonarSource/sonar-java/pull/890/files#diff-9084a9c491d851a91e1c344cac8337fcR58))